bgpd/bgp_zebra.c

```diff
@@ -1652,15 +1652,25 @@ int bgp_redistribute_resend(struct bgp *bgp, afi_t afi, int type,
 }
 
 /* Redistribute with route-map specification.  */
-int bgp_redistribute_rmap_set(struct bgp_redist *red, const char *name)
+int bgp_redistribute_rmap_set(struct vty *vty, struct bgp_redist *red,
+			      const char *name)
 {
+	struct route_map *route_map = route_map_lookup_by_name(name);
+
+	if (!route_map) {
+		vty_out(vty,
```

Comment:
Why can't we print the route map name in vty message as well ?

Reply:
we can 🙂